Kernel/Arch/aarch64/kprintf.cpp

```diff
@@ -64,3 +64,26 @@ void kernelearlyputstr(char const* characters, size_t length)
     for (size_t i = 0; i < length; ++i)
         console_out(characters[i]);
```

Comment:
Should this just call dbgputstr now?

Reply:
I've made all the function call `kernelputstr` instead. This code is temporarily anyway, since it should be merged with `Kernel/kprintf.cpp`.